### ISSUE TITLE
Add brakeman

### ISSUE THREAD

**Issue by composerinteralia** [thoughtbot/suspenders#1035]
Brakeman scans for security vulnerabilities. I think we should add it to the default Gemfile so folks are encouraged to run it locally. Once we have a better CI setup, it would also be nice to run this by default on CI.

**Comment by thiagoa** [comment]
+1 to this!

**Comment by stevepolitodesign** [comment]
This would pair well with #1138. This is also the default in the next release of Rails https://github.com/rails/rails/pull/50507

**Comment by stevepolitodesign** [comment]
I don't think this is relevant anymore thanks to the [latest release (https://github.com/thoughtbot/suspenders/releases/tag/v20251219.0).

Rails ships with Brakeman and a CI task, and Suspenders picks that up.
